movies/src/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import {createStore, applyMiddleware} from 'redux';
+import ReduxPromise from 'redux-promise';
+import {BrowserRouter, Route} from 'react-router-dom';
+import './index.css';
+import App from './App';
+import reducer from './reducers';
+import Movies from './components/Movies';
+import Synop from './components/Synop';
+
+const createStoreWithMiddleware = applyMiddleware(ReduxPromise)(createStore);
+
+ReactDOM.render(<Provider store={createStoreWithMiddleware(reducer)}>
+    <BrowserRouter>
+        <div>
+        <Route exact path="/" component={App} />
```

Comment:
Indent all of the `Route` tags once more so that they're clearly nested inside the `div` tags.